### DISCUSSION TITLE
Timezone Change Event

### DISCUSSION THREAD

**Discussion by rhaynel2024** [gdelataillade/alarm#253] [Ideas]
Alarm is very critical issue might happen to the people who travel alot, Can you add the timezone feature to auto reschedule the alarms based on the new user timezone.

**Comment by gdelataillade** [comment]
Hi @rhaynel2024 

It is indeed an important feature to add. Right now I'm focusing on making the package as bug-free as possible and then I'll probably work on timezone change support, but it might take a while because it's pretty complex. I'll keep you updated here: #164
